src/builder/index.js

```diff
@@ -7,6 +7,7 @@ import { bundle } from 'builder/bundle'
 import { watch } from 'builder/watcher'
 import { source, sourcePath, destinationPath } from 'utils'
 import { log, warn, startSpinner, stopSpinner } from 'utils/logger'
+import * as Util from "node/util";
```

Comment:
import should be done as follows

```suggestion
import { inspect }  from 'util'
```

Reply:
ok no problem for me :) 
I've committed your suggested changes